src/main/java/org/folio/circulation/resources/CirculationRulesResource.java

```diff
@@ -138,4 +233,11 @@ private static void circulationRulesError(HttpServerResponse response, DecodeExc
     body.put("message", e.getMessage());  // already contains line and column number
     new JsonResponseResult(422, body, null).writeTo(response);
   }
+
+  private static class PolicyEntity extends Policy {
```

Comment:
What is a policy entity?

Reply:
I just need some common entity for mapping from json. But we can't use Policy because of the Policy class is abstract. So I created PolicyEntity.

Reply:
I don't really follow.

Is the intent to load all of the policies regardless of their type? If so, was making policy concrete rather than abstract considered? I don't really understand how `PolicyEntity` is any different to that.

If there is a desire for a separate class for only the `id` and `name`, then I suggest keeping it entirely separate from the rest of the policy modelling.

Reply:
Got it, thanks.
It's really not needed.
Updated